src/main/content/index.html

```diff
@@ -215,57 +215,122 @@ <h3 id="logos_title">{% t home.openlibery_opensource_community_title %}</h3>
     <h2 id="news_title">{% t home.openliberty_news %}</h2>
     <p id="news_text">{% t home.openliberty_news_text %}</p>
     <div class="row">
-        <div class="col-xs-12 col-lg-12">
-            <div id="blog_post">
-                {% for post in site.posts limit:1 %}
-                    {% if post.redirect_link %}
-                        <h2 class="blog_post_title"><a href="{{ post.redirect_link }}" target="_blank" rel="noopener" class="blog_post_title_link">{{ post.title | escape }}</a>
-                          <a href="{{ post.redirect_link }}" target="_blank" rel="noopener" class="blog_post_title_link">  
-                          <span class="continued_text">
-                                {% assign continue_text = ' [Continued on ' %}
-                                {% assign domain_name = post.redirect_link | remove: "https://" | remove: "http://" | remove: "www." | append: "]" %}
-                                {{ continue_text | append: domain_name }}
-                            </span>
-                            </a>
-                        </h2>
+        <!-- Blog without release -->
+        <div class="col-xs-12 col-lg-12" id="blog_col">   
```

Comment:
Please use a class if there are 2 blog_col on the page since id should be unique. You could also change the class to col-lg-6 and it will split the columns half between the maximum of 12 per the Bootstrap grid system: https://getbootstrap.com/docs/4.0/layout/grid/